polar-core/src/parser.rs

```diff
@@ -32,65 +32,60 @@ pub enum Line {
 }
 
 fn to_parse_error(e: ParseError<usize, lexer::Token, error::ParseError>) -> error::ParseError {
+    use error::ParseError::*;
```

Comment:
> Also why isn't this just an Into implementation?

This actually works better. You end up with `parse_things(...).map_err(to_parse_error)...`

instead of `.map_err(std::into::Into::into)` or `.map_err(|e| e.into())`.

Since we wouldn't see any parse errors outside of this file, the value of implementing a trait seems low.

Reply:
> I find this function confusing because of all the enum variants with the same names but from different namespaces, it was less confusing before when the `error::` namespace on the RHS was more explicit.

9ae45835